packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -329,32 +315,27 @@ export default class ImageEdit implements EditorPlugin {
      * Create editing wrapper for the image
      */
     private createWrapper(operation: ImageEditOperation | CompatibleImageEditOperation) {
-        // Wrap the image with an entity so that we can easily retrieve it later
-        const { wrapper } = insertEntity(
-            this.editor,
-            IMAGE_EDIT_WRAPPER_ENTITY_TYPE,
-            wrap(this.image, KnownCreateElementDataIndex.ImageEditWrapper),
-            false /*isBlock*/,
-            true /*isReadonly*/
-        );
+        //Clone the image and insert the clone in a entity
+        this.clonedImage = this.image.cloneNode(true) as HTMLImageElement;
+        const wrappedImage = wrap(this.clonedImage, KnownCreateElementDataIndex.ImageEditWrapper);
+        this.wrapper = wrap(wrappedImage, KnownCreateElementDataIndex.ImageEditContainer);
```

Comment:
Actually, I mean merge these two into one. Because wrappedImage is only used here. So we can do
this.wrapper = wrap(this.clonedImage, KnownCreateElementDataIndex.ImageEditWrapper);

Then you can modify the definition of ImageEditWrapper to make it has two layers of wrapper.